CMakeLists.txt

```diff
@@ -89,6 +89,13 @@ if(BUILD_UNIVERSAL_CLIENT)
   set(client-only ON)
 endif()
 
+option(BUILD_WEBUI "Build with WEBUI" ON)
+if(BUILD_WEBUI)
+  set(ENABLE_WEBUI ON)
+else()
+  set(ENABLE_WEBUI OFF)
+endif()
+
```

Comment:
```suggestion
option(ENABLE_WEBUI "Build with WEBUI" ON)
```

Reply:
Change request applied in https://github.com/bareos/bareos/pull/1213/commits/3ee9e618a1ec92c7947bb318f2ef81cabff8f643 and https://github.com/bareos/bareos/pull/1213/commits/11fbdd2ae4c315d7f76c71b0287ad1b3c69eaa21